apps/desks.py

```diff
@@ -675,7 +680,7 @@ def _users_aggregation(self, desk_id: str) -> None:
         overview = []
         for a in users_aggregation:
             role = a['_id']
-            authors_dict = {}
+            authors_dict: Dict[str, Any] = {}
```

Comment:
I think this syntax is not available in Python 3.5, that's one of the reason I want to get rid of it. If this PR is for Superdesk 2.x, it may not be a pb though. I won't repeat this comment, but it also applies in other locations in this patch.

Reply:
@jerome-poisson what syntax? Type definition and variable initialization in one row?

Reply:
@jerome-poisson just marked this pr to 2.1

Reply:
type hint in random assignment, IIRC Python 3.5 only supports type hints in functions/methods definitions, class attributes and other variable can only be typed from Python 3.6+

Reply:
@jerome-poisson indeed https://www.python.org/dev/peps/pep-0526/
but this feature (mypy) is for SD 2.1, so we are fine ;)